### ISSUE TITLE
Looking forward to releasing

### ISSUE THREAD

**Issue by shenfe** [ptaszynski/yacis-corpus#1]
Hi, any plan to release this corpus? It looks great!

**Comment by ptaszynski** [comment]
Thanks for the message. Yes. We are delayed by licensing stuff and reviews for the accompanying paper, but the plan is to have everything dumped by the end of March 2023.

**Comment by okuta** [comment]
I'm interested in this dataset.
When is it accessible?

**Comment by ptaszynski** [comment]
Yes. As soon as I find time to properly wrap everything up and submit a data descriptor about it it will be released.

**Comment by okuta** [comment]
Thanks, I'm looking forward to it!
